1.3/spec.md

```diff
@@ -0,0 +1,359 @@
+# MBTiles 1.3
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in
+this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+## Abstract
+
+MBTiles is a specification for storing tiled map data in
+[SQLite](http://sqlite.org/) databases for immediate usage and for transfer.
+MBTiles files, known as **tilesets**, MUST implement the specification below
+to ensure compatibility with devices.
+
+## Database Specifications
+
+Tilesets MUST be valid SQLite databases of
+[version 3.0.0](http://sqlite.org/formatchng.html) or higher.
+Only core SQLite features are permitted; tilesets MUST NOT require extensions.
+
+MBTiles databases SHOULD use [the officially assigned magic number](http://www.sqlite.org/src/artifact?ci=trunk&filename=magic.txt)
+to be easily identified as MBTiles.
+
+## Database
+
+Note: the schemas outlined are meant to be followed as interfaces.
+SQLite views that produce compatible results are equally valid.
+For convenience, this specification refers to tables and virtual
+tables (views) as tables.
+
+### Metadata
+
+#### Schema
+
+The database MUST contain a table or view named `metadata`.
+
+This table or view MUST yield exactly two columns named `name` and
```

Comment:
We should specify the types of the columns here.

Reply:
Thanks @flippmoke. This and your other comments should be fixed in dc659f4